CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
 # CHANGELOG
 
-## Master
+## v1.0.0
 
+### Upgrade Considerations
+- Panel Block, Panel List, Grey Panel Block, Promo Block, Promo Group Block have been removed from the plugin, so references to these in your project will need to be removed in order to upgrade. Ensure to make migrations after doing so.
```

Comment:
We should probably refer to these blocks by their python class name (`PanelBlock`, `PanelList` etc.)